src/screens/WebsocketResearchScreen/SampleChatScreen.tsx

```diff
@@ -24,22 +28,23 @@ export const styles = StyleSheet.create({
     flexDirection: 'column',
     height: height - 85
   },
+  flatlistContainer: {
+    paddingTop: 5,
+    paddingBottom: 20
+  },
   chatContainer: {
     display: 'flex',
     height: '100%'
   },
   inputContainer: {
     backgroundColor: colors.white,
-    position: 'absolute',
-    bottom: 0,
-    // height: 50,
-    left: 0,
-    right: 0,
     zIndex: 100,
     padding: 8,
-    paddingBottom: 16,
     borderTopColor: colors.lightgrey,
-    borderTopWidth: 1
+    borderTopWidth: 1,
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'center'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new styles for `flatlistContainer` and `inputContainer` seem to be fine. However, it's important to note that the `position: 'absolute'` style has been removed from `inputContainer`. This could potentially affect the layout of your chat screen, especially if there are other components that rely on this container being positioned absolutely. If you notice any layout issues after this change, you might want to consider reintroducing absolute positioning.



<!-- This is an auto-generated comment by OSS CodeRabbit -->